importer/api/__init__.py

```diff
@@ -130,9 +132,9 @@ def import_children(course, element, parent):
     of an XML tree.
 
     Args:
-        course (learningresources.Course): Course
+        course (Course): Course
```

Comment:
this should be `learningresources.models.Course` since `Course` isn't imported here